CMakeLists.txt

```diff
@@ -14,6 +14,10 @@ project(tt-metal
 find_package(Boost REQUIRED COMPONENTS thread filesystem system regex)
 find_package(GTest REQUIRED)
 find_package (Python3 COMPONENTS Interpreter Development)
+find_library(NUMA_LIBRARY NAMES numa)
+if (NOT NUMA_LIBRARY)
```

Comment:
does the `REQUIRED` parameter not work here?

Reply:
I would also prefer `REQUIRED`